vms/platformvm/service_test.go

```diff
@@ -217,7 +217,6 @@ func TestGetTxStatus(t *testing.T) {
 		},
 	}))
 
-	oldSharedMemory := mutableSharedMemory.SharedMemory
```

Comment:
Why was this test modified?

Reply:
Because we cache tx verification status over the rpc (as we do on the X-chain). There is a TODO to decide on if we want to re-verify in these situations.